doc/howto_release.md

```diff
@@ -1,15 +1,57 @@
 # How to release GRASS GIS binaries and source code
 
-*Note: Some steps in this text are to be done by the development coordinator
+## Assumptions
+
+- You have communicated with other developers, e.g., through grass-dev mailing list.
+- You have communicated with a development coordinator.
+- You have already cloned the repo with Git.
+- Your own fork is the remote called "origin".
+- The OSGeo repo is the remote called "upstream".
+- You don't have any local un-pushed or un-committed changes.
+- You are using Bash or a similar shell.
+
+*Note: Some later steps in this text are to be done by the development coordinator
 (currently Markus Neteler and Martin Landa) due to needed logins.*
 
-## HOWTO create a release
+## Prepare the local repo
+
+Update your remotes and switch to branch:
+
+```bash
+git fetch --all --prune && git checkout releasebranch_8_2
+```
+
+Confirm that you are on the right branch and have no local changes:
+
+```bash
+# Should show no changes:
+git status
+# Should give no output at all:
+git diff
+git diff --staged
+git log upstream/releasebranch_8_2..HEAD
+# Should give the same as last commits visible on GitHub:
+git log --max-count=5
+```
+
+Now you can merge (or rebase) updates from the remote your local branch
+and optionally update your own fork:
+
+```bash
+git merge upstream/releasebranch_8_2 && git push origin releasebranch_8_2
+```
+
+Use _git log_ and _git show_ to verify the result:
```

Comment:
```suggestion
Use `git log` and `git show` to verify the result:
```

Reply:
Changed the emphasis with underscores to code with backticks in 01336aa155, but it is actually questionable. While the code block which immediately follows is code to be executed, the names in the text refer to the tools rather than representing the code to be executed and in that context emphasis (italics) is commonly used these days I think. Both however happen to be meaningful code as well.